src/tooling/piral-cli/src/common/config.ts

```diff
@@ -41,6 +42,22 @@ export interface PiralCliConfig {
    * Sets the schema version to be used for pilets.
    */
   schemaVersion?: PiletSchemaVersion;
+  /**
+   * Automatically open the browser.
+   */
+  openBrowser?: boolean;
+  /**
+   * Port number.
+   */
+  port?: number;
+  /**
+   * Template language.
+   */
+  SourceLanguage?: SourceLanguage.ts;
```

Comment:
Maybe call it `language` or `sourceLanguage` (lower cased `s`) for consistency.

Reply:
Fixed.